ruby_03-professional_rails_applications/building-a-vps.markdown

```diff
@@ -352,7 +360,7 @@ We're looking for the settings for Phusion Passenger. If you're using Vim, you c
 ##
 
 passenger_root /usr/lib/ruby/vendor_ruby/phusion_passenger/locations.ini;
-passenger_ruby /home/deploy/.rvm/gems/ruby-2.2.1/wrappers/ruby;
+passenger_ruby /home/deploy/.rvm/wrappers/ruby-2.2.1/ruby;
```

Comment:
The original line did not work for us, so you had us change the line to the one I added. I don't fully understand the difference, but this change was one you made on the fly.